fcl/dunefdvd/gen/solar/prodmarley_flat_dunevd10kt_1x8x14_3view_30deg.fcl

```diff
@@ -0,0 +1,5 @@
+#include "prodmarley_nue_spectrum_dunevd10kt_1x8x14_3view_30deg.fcl"
```

Comment:
Inherits from SN fcl.   This may just need to be changed later on.

Reply:
What if it inherits a common fhicl from the supernova folder and not a geo specific one?

Reply:
@mantheys could we rename these solar fhicls? I think the difference between this file `prodmarley_flat_dunevd10kt_1x8x14_3view_30deg.fcl` and the one in the supernova folder `prodmarley_nue_flat_dunevd10kt_1x8x14_3view_30deg.fcl` needs to be clearer in the name. I was thinking of prodmarley_**solar_**flat_dunevd10kt_1x8x14_3view_30deg.fcl. What do you think?

Reply:
Yeah inheriting from a common fcl can absolutely work.  Alternatively, we could setup the default Marley configuration upstream to use these params.  The main point is that there would be a single point of upstream maintenance.

Reply:
Ok, I am going with the common fhicl for now and we can discuss this further.